Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -553,7 +553,7 @@
         end
         
         % Initialize matrices with NaNs

```

Comment:
To avoid confusion, probably best to replace 

`% Initialize matrices with NaNs`

with

`% Initialize output data matrix ([x y] = [subject/session ROI-statistics]) with NaNs`

Reply:
Done